library/modules/Gui.cpp

```diff
@@ -130,529 +130,307 @@ static std::string getNameChunk(virtual_identity *id, int start, int end)
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.location_details.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/LocationDetails";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.hauling_stop_conditions.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/HaulingStopConditions";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.assign_vehicle.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/AssignVehicle";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.stockpile_link.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/StockpileLink";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.stockpile_tools.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/StockpileTools";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.custom_stockpile.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/CustomStockpile";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.create_squad.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/CreateSquad";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.squad_supplies.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/SquadSupplies";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.assign_uniform.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/AssignUniform";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.create_work_order.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/CreateWorkOrder";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.hotkey.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/Hotkey";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.options.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/Options";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.help.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/Help";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.settings.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/Settings";
+        focusStrings.push_back(newFocusString);
+    }
+    if (game->main_interface.squad_equipment.open) {
+        newFocusString = baseFocus;
+        newFocusString += "/SquadEquipment";
+        focusStrings.push_back(newFocusString);
     }
-*/
-}
-
-/* TODO: understand how this changes for v50
-DEFINE_GET_FOCUS_STRING_HANDLER(dungeonmode)
```

Comment:
you should probably leave this in but commented out for the time when dungeonmode (adventure mode) rises again.